x-pack/plugins/siem/public/mock/mock_timeline_data.ts

```diff
@@ -28,10 +28,10 @@ export const mockTimelineData: TimelineItem[] = [
         action: ['Action'],
         category: ['Access'],
         module: ['nginx'],
-        severity: ['3'],
+        severity: [3],
       },
-      source: { ip: ['192.168.0.1'], port: ['80'] },
-      destination: { ip: ['192.168.0.3'], port: ['6343'] },
+      source: { ip: ['192.168.0.1'], port: [80] },
+      destination: { ip: ['192.168.0.3'], port: [6343] },
       user: { id: ['1'], name: ['john.dee'] },
```

Comment:
Should these user.id's be a number? Looks like they are currently in feature-secops.

Reply:
According to the docs:
https://www.elastic.co/guide/en/ecs/current/ecs-user.html

That is a `keyword` so I think it is typed correctly. The tests are just a bit misleading as they use the set of natural numbers